docs/DEVELOPMENT.md

```diff
@@ -12,82 +12,109 @@ install -m 755 operator-sdk_linux_amd64 ${SOME_DIR_IN_YOUR_PATH}/operator-sdk
 ```
 - podman, podman-docker or docker
 - Access to OpenShift cluster (4.12+)
-- Container registry to storage images
+- Container registry to store images
 
 ### Get a token on registry.ci.openshift.org
 Our builder and base images are curated images from OpenShift.
 They are pulled from registry.ci.openshift.org, which require an authentication.
 To get access to these images, you have to login and retrieve a token, following [these steps](https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-log-in-to-pull-images-that-require-authentication)
 
+##### Details on token
+- You should go to _app.ci_ in the above doc, login with SSO
+- Go to _copy login_ on your _username_ in the top right
+- You will login and see _view token_.  Click on that to see
+- **Your API token is**.  The next line is your full token
+- _podman login_ with your username and the token as your password should put the token in your AUTH_FILE
+
+#### Using the token
+If docker build does not automatically use the token in the default authfile, it can be forces:
+```shell
+export AUTH_FILE='~/.docker/config.json'
```

Comment:
From the `podman-login(1)` manual page :
```
The default path for reading and writing credentials is ${XDG_RUNTIME_DIR}/containers/auth.json.
Podman uses existing credentials if the user does not pass in a username.  Podman first searches
for  the  username and  password  in  the ${XDG_RUNTIME_DIR}/containers/auth.json,  if  they are
not valid, Podman then uses any existing credentials found in $HOME/.docker/config.json.
```

```suggestion
export AUTH_FILE="${XDG_RUNTIME_DIR}/containers/auth.json"
```

Reply:
Hmm... I gave a try without setting `AUTH_FILE` and it worked :
- `podman login` does add the credentials to `"${XDG_RUNTIME_DIR}/containers/auth.json"` 
- `podman build` does find the credentials there
I now suspect that `AUTH_FILE` isn't needed at all.

What were the steps that ended up in `docker build does not automatically use the token in the default authfile` ?